integrations/amazon_bedrock/src/haystack_integrations/components/generators/amazon_bedrock/chat/adapters.py

```diff
@@ -271,6 +271,158 @@ def _to_anthropic_message(self, m: ChatMessage) -> Dict[str, Any]:
         return {"content": [{"type": "text", "text": m.content}], "role": m.role.value}
 
 
+class MistralChatAdapter(BedrockModelChatAdapter):
+    """
+    Model adapter for the Mistral chat model.
+    """
+
+    chat_template = (
```

Comment:
Can we add tests for the logic inside this template? Also we should probably move to triple quotes to ease copy/paste of the template code and readability

Reply:
Sure, good point on both!